packages/legacy-scripting-runner/bundle.js

```diff
@@ -186,14 +183,20 @@ const addResponse = (event, bundle, convertedBundle) => {
 
 // Convert bundle from CLI to WB based on which event to run
 const bundleConverter = async (bundle, event, z) => {
-  let defaultMethod = 'GET';
-
-  if (
-    event.name.startsWith('create') ||
-    event.name.startsWith('auth.oauth2') ||
-    event.name.startsWith('trigger.hook.subscribe')
-  ) {
-    defaultMethod = 'POST';
+  let requestMethod = _.get(bundle, 'request.method');
+  if (!requestMethod) {
+    if (
+      event.name.startsWith('create') ||
+      event.name.startsWith('auth.oauth2') ||
+      event.name.startsWith('trigger.hook.subscribe')
+    ) {
+      requestMethod = 'POST';
+    } else if (event.name === 'trigger.hook.unsubscribe.pre') {
+      requestMethod = 'DELETE';
+    }
+  }
+  if (!requestMethod) {
```

Comment:
minor style thing, but couldn't this be moved a few lines up as an `else` clause? I think the logical flow is the same.